### ISSUE TITLE
NVDA is not announcing the label name in Browse mode.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#14962]
**Steps to reproduce:**

1. Open [Codepen for checkbox](https://codepen.io/Rudra-Jairam/pen/ZEqdrYj) in Edge browser.
2. Turn on NVDA and navigate to the checkbox control 'Hello' in browse mode using down arrow key.
3. Observe the screen reader behavior.

**Actual behavior:**
In Browse mode, when focus lands on the checkbox control by pressing down arrow key, NVDA is not reading out the label name. NVDA just announces checkbox not checked.

**Expected behavior:**
In Browse mode, when focus lands on the checkbox control by pressing down arrow key, NVDA should announce its label name 'Hello' including it's state whether checked or not.

**System configuration**
NVDA installed/portable/running from source: Installed
NVDA version: NVDA version 2023.1
Windows version: Edge Version 114.0.1823.37 (Official build) (64-bit)
Other information about your system: OS Version: 22H2 OS Build 22621.1702
Does the issue still occur after restarting your computer? yes

<img width="487" alt="Hello checkbox" src="https://github.com/nvaccess/nvda/assets/93735775/2a9d2cbd-9c3d-492d-bcb1-4fad49290727">

**Comment by seanbudd** [comment]
This is intentional behaviour, you can read the label when arrowing down again
